zerver/models.py

```diff
@@ -1692,18 +1692,28 @@ class Referral(models.Model):
     email = models.EmailField(blank=False, null=False) # type: Text
     timestamp = models.DateTimeField(auto_now_add=True, null=False) # type: datetime.datetime
 
-class ScheduledJob(models.Model):
-    scheduled_timestamp = models.DateTimeField(auto_now_add=False, null=False) # type: datetime.datetime
+class ScheduledEmail(models.Model):
+    # Exactly one of to_user or to_address should be set
+    to_user = models.ForeignKey(UserProfile, null=True, on_delete=CASCADE) # type: UserProfile
+    # Just the address part of a full "name <address>" email address
+    to_address = models.EmailField(null=True) # type: Text
+
     type = models.PositiveSmallIntegerField() # type: int
-    # Valid types are {email}
-    # for EMAIL, filter_string is recipient_email
-    EMAIL = 1
+    # Valid types are below
+    FOLLOWUP = 1
+    DIGEST = 2
+    INVITATION_REMINDER = 3
 
-    # JSON representation of the job's data. Be careful, as we are not relying on Django to do validation
+    scheduled_timestamp = models.DateTimeField() # type: datetime.datetime
```

Comment:
I'm not sure why it was there; `auto_now_add=False` is the default value for `models.DateTimeField`.